src/controllers/user/routes.ts

```diff
@@ -9,4 +10,11 @@ userRouter.route('/')
 .post( validationHandler( validation.create ), UserController.create)
 .put(validationHandler(validation.update ), UserController.update);
  userRouter.route('/:id').delete(validationHandler( validation.delete ), UserController.delete);
+
+//  userRouter.route('/me')
+//  .get(authMiddleware ( validation.get, 'all'), UserController.me);
+ userRouter.route('/me');
+ // .get(authMiddleware ( validation.get, 'all'), UserController.me);
```

Comment:
Uncomment this like, else how  UserCpontroller.me function get called on route "/me"